lib/modules/onc_program_procedure.yml

```diff
@@ -0,0 +1,1219 @@
+          associated with the profiles specified in the standard adopted in
+          § 170.213 and implementation specification adopted in
+          § 170.215(a)(2), including:
+          * “AllergyIntolerance”;
+          * “CarePlan”;
+          * “CareTeam”;
+          * “Condition”;
+          * “Device”;
+          * “DiagnosticReport”;
+          * “DocumentReference”;
+          * “Encounter”;
+          * “Goal”;
+          * “Immunization”;
+          * “Location”;
+          * “Medication” (if supported);
+          * “MedicationRequest”;
+          * “Observation”;
+          * “Organization”;
+          * “Practitioner”;
+          * “PractitionerRole”;
+          * “Procedure”; and
+          * “Provenance”.
+        inferno_supported: partial
+        inferno_notes: |
+          If the patient API uses a separate FHIR endpoint than the practitioner-facing API,
+          we do not test the second order (non-USCDI) resources because our tests are not
+          sophisticated enough in this area to find references to check.  Specifically:
+          Organization, Practioner, PractitionerRole.  We overtest for Encounter on the patient API
+          tests.  For the pratitioner EHR launch, we may not require all scopes anywhere explicitly,
+          though we do test for all resources which implies that we require all scopes.
+        inferno_tests:
+          - SPB-OSLS-01 - SPB-OSLS-08
+          - EHB-OELS-01 - EHB-OELS-10
+          - AVU-01 - AVU-12
+          - USCP-08
+          - USCAI-03
+          - USCCP-05
+          - USCCT-02
+          - USCC-06
+          - USCID-03
+          - USCDRRN-07
+          - USCDRLRR-07
+          - USCDR-08
+          - USCG-04
+          - USCI-04
+          - USCMR-05
+          - USCSSO-06
+          - USCPWHO-06
+          - USCLRO-06
+          - USCPBMIAO-06
+          - USCPO-06
+          - USCOBH-06
+          - USCOBT-06
+          - USCOBP-06
+          - USCOBW-06
+          - USCOHC-06
+          - USCOHR-06
+          - USCORR-06
+          - USCPROC-05
+          - USCL-01
+          - USCO-01
+          - USCPR-01
+          - USCPRO-01
+          - USCPROV-01
+      - index: 10 
+        SUT: |
+          [Both] The health IT developer demonstrates the ability of the
+          Health IT Module to deny an application’s authorization request
+          according to a patient’s preferences selected in step 9 of this
+          section in accordance with the implementation specification
+          adopted in § 170.215(a)(3).
+        TLV: |
+          [Both] The tester verifiesthe ability of the Health IT Module to
+          deny an application’s authorization request according to a
+          patient’s preferences selected in step 9 of this section in
+          accordance with the implementation specification adopted in §
+          170.215(a)(3).
+        inferno_supported: partial
+        inferno_notes: |
```

Comment:
Similar to the previous test, what needs to be demonstrated?  Are you envisioning a user denying some or all of the resources, and we just check to see what scopes are actually granted at the end of the launch?  Or do we then need to check to see if the access token given matches the scopes granted by trying to access all of the resource types and seeing what happens?  If the former, then we do this.  If the latter, then we mostly do this, but there is a small gap for Location/Organization/Practioner/PractionerRole because it requires a bit more work to find IDs for us to check.